pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -591,25 +630,15 @@ func TestGetTaskFunc_RemoteResolution(t *testing.T) {
 			"apiVersion: tekton.dev/v1",
 			remoteTaskYamlWithoutDefaults,
 		}, "\n"),
-		wantTask: parse.MustParseV1beta1Task(t, `
-metadata:
-  name: simple
-  namespace: default
-spec:
-  steps:
-  - image: something
-  params:
-  - name: foo
-    type: string
-`),
+		wantTask: parse.MustParseV1Task(t, remoteTaskYamlWithoutDefaults),
```

Comment:
what's the reason for this change? Is it just a cleanup?

Reply:
I think this is because when we are converting v1beta1 Params to v1 Params we are adding the default `string` type for the empty "" type, while when we swap to v1, we do not need such default added in v1. The related PR is https://github.com/tektoncd/pipeline/pull/5625.